recipes/outrigger/meta.yaml

```diff
@@ -0,0 +1,88 @@
+package:
+  name: outrigger
+  version: "1.0.0"
+
+source:
+  fn: outrigger-1.0.0.tar.gz
+  url: https://pypi.python.org/packages/13/b9/02d6db3140448685a9651918e3c2831401983d8123dd2560320283abddcc/outrigger-1.0.0.tar.gz
```

Comment:
Please use the something like the following url for `pypi`:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18